libs/core-ui/src/lib/Context/ModelAssessmentContext.tsx

```diff
@@ -59,6 +59,18 @@ export interface IModelAssessmentContext {
     request: any,
     abortSignal: AbortSignal
   ) => Promise<IHighchartBoxData>;
+  requestGlobalCausalEffects?: (
```

Comment:
We're accumulating rather many requestXYZ functions here. Is there perhaps a way we might bundle them together?
We might otherwise end up with dozens of these as input params. 

@xuke444 any ideas? Off the top of my head, I guess we could define a class that has all these as params, but that is also not particularly pretty.

Reply:
Chatted with @xuke444 briefly on this and he agrees there should be some structure. Perhaps a tree (like a dict of dicts?). Wdyt @gaugup ?

Reply:
Sure, I am happy to introduce such a structure. Let's brainstorm how to put a structure. Nevertheless, the aforementioned structure is not in scope of this PR.

Reply:
I have created a work item for this ask internally.